release_notes.md

```diff
@@ -3,6 +3,11 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Update Python Worker Version to [4.28.1](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.28.1)
 - DotnetIsolated worker artifact clean up (#9976)
```

Comment:
Can we add OTel support info in Release notes?

Reply:
Not now, we are still trying to figure out release announcement for OTel.